python/pyxet/pyxet/sync.py

```diff
@@ -118,6 +123,11 @@ def _sync_with_info(self, executor, futures, src_path, dest_path):
         total_size = 0
         for abs_path, src_info in self._src_fs.find(src_path, detail=True).items():
             relpath = _rel_path(abs_path, src_path)
+
+            if relpath == '.':
```

Comment:
Do we also need to check for `''` and `'..'`? I don't expect those to occur, but there could be some fsspec that does that?